Douban.js

```diff
@@ -229,13 +233,9 @@ function doWeb(doc, url) {
 		scrapeAndParse(doc, url);
 	}
 }
+
 /** BEGIN TEST CASES **/
 var testCases = [
-	{
-		"type": "web",
-		"url": "http://book.douban.com/subject_search?search_text=Murakami&cat=1001",
-		"items": "multiple"
-	},
```

Comment:
Is that not anymore working or why did you delete it?

Reply:
It's not working. It's using some delayed loading technique, I can't solve it yet.

Reply:
Okay, I will add a note about this.